command/src/main/java/com/jagrosh/jdautilities/command/impl/CommandClientImpl.java

```diff
@@ -607,6 +674,25 @@ else if(event.isFromType(ChannelType.PRIVATE) || event.getTextChannel().canTalk(
             listener.onNonCommandMessage(event);
     }
 
+    private void onSlashCommand(SlashCommandEvent event)
```

Comment:
```suggestion
    @Override
    private void onSlashCommand(SlashCommandEvent event)
```

Reply:
The overrides only appear to apply when you extend ListenerAdapter, not EventListener that CommandClientImpl does 🤔 I can't exactly tell why or what benefits it has not doing it